src/components/diagrams/singleLineDiagram/single-line-diagram-content.js

```diff
@@ -398,15 +405,14 @@ const SingleLineDiagramContent = forwardRef((props, ref) => {
                             error: error.message,
                             svgUrl: props.svgUrl,
                         });
-                        let msg;
-                        if (error.status === 404) {
-                            msg = `Voltage level not found`;
+                        // if svg is not found, an empty SLD with a warning message is now displayed. No need to show snackError.
+                        if (error.status !== 404) {
+                            snackError({
+                                messageTxt: error.message,
+                            });
                         } else {
-                            msg = error.message;
+                            handleWarning(props.diagramId, props.svgType);
```

Comment:
Parameters are not needed here, and maybe the function could be called "handleMissingEquipment" or something like that

Reply:
done